management/storageservice/client.go

```diff
@@ -92,7 +92,7 @@ func (self StorageServiceClient) CreateStorageService(name, location string) (*S
 		return nil, fmt.Errorf(errParamNotSpecified, "location")
 	}
 
-	storageDeploymentConfig := self.createStorageServiceDeploymentConf(name, location)
+	storageDeploymentConfig := self.createStorageServiceDeploymentConf(name, location, AccountTypeStandard_LRS)
```

Comment:
The code that was there never touched the properties that were deprecated, serialized the 'false' values anyway. (API returns 400)
The StorageServiceDeployment struct is simple enough, shall I just make that the input of CreateStorageService?